src/pcost.py

```diff
@@ -0,0 +1,29 @@
+import csv
+
+
+def portfolio_cost(filename: str) -> float:
```

Comment:
I will be picky but in general `filename` refers to the name of the file and not the full path. Here you are providing the full path (otherwise you can't open it without knowing the directory) so you should update the name of the arguments to `file_path`:
```suggestion
def portfolio_cost(file_path: str) -> float:
```